src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -1,5 +1,9 @@
 <div
-  class="c-mini-shelf"
+  class="c-mini-shelf
+{{ showOnAirBorder && !showNextBorder ? 'c-mini-shelf--on-air' : '' }}
+{{ !showOnAirBorder && showNextBorder ? 'c-mini-shelf--next' : '' }}
+{{ showOnAirBorder && showNextBorder ? 'c-mini-shelf--on-air-and-next' : '' }}
+{{ !showOnAirBorder && !showNextBorder ? 'c-mini-shelf--normal' : '' }}"
```

Comment:
You can simplify this a lot by writing:
![image](https://github.com/tv2/ng-sofie/assets/23049647/07ceba14-67ea-4446-a4a5-4758b4955293)

Then, in your CSS you can write:
![image](https://github.com/tv2/ng-sofie/assets/23049647/aa75dbb6-9eab-4407-bbb6-4fa6ef6c1860)

Notice how the `c-mini-shelf-normal` is gone?
You use it set the default border of the MiniShelf, so that can just be an attribute on `c-mini-self`:
![image](https://github.com/tv2/ng-sofie/assets/23049647/869589e6-2f31-4fb6-b435-4940934b1590)


CSS selectors works on a priority system. The more specific the selector, the higher the priority and we can utilize that to write less CSS. If the priority is equal, then it is just whatever has been read last. Hence `.next` is above `.on-air` in my example so `.on-air.next` only need to set the right border.


Note: The names I selected for my CSS classes is mainly meant for demonstrating purposes.